test/jdk/java/net/httpclient/whitebox/java.net.http/jdk/internal/net/http/SSLTubeTest.java

```diff
@@ -144,6 +146,7 @@ private void clientReader() {
                     publisher.submit(List.of(bb));
                 }
             } catch (Throwable e) {
+                System.out.println("clientReader got exception: " + e);
```

Comment:
Hello Daniel, I think since we are printing the stacktrace on the next line, it might be better to use `System.err.println` here so that this message to ends up in the `System.err` section like the stacktrace. Same suggestion for 2 more places in this PR where we are introducing such messages in the Throwable block.

Reply:
I wanted to print this on System.out by design - to avoid the trace being lost among the many debug traces on System.err. What I'm interested mostly here is knowing whether the thread exited normally or with an error.

Reply:
> If you believe it's important enough I believe we could write both on System.out and System.err, but I specifically wanted it on System.out.

The current way you have it is fine. I didn't know the use of `System.out` was intentional.